### PR TITLE
We can now update an incident

### LINKED ISSUE #214: Add option to edit Incidents
Currently not have an option to edit Incidents.

### PR DESCRIPTION
**Pull request by Crash--**
I'm not an expert of Laravel, but now you can edit an incident. 

Should close #214

### CODE REVIEW

File: src/Http/Controllers/DashIncidentController.php [hunk 1]
```diff
@@ -21,8 +21,8 @@ public function showIncidents()
         $incidents = Incident::orderBy('created_at', 'desc')->get();
 
         return View::make('dashboard.incidents.index')->with([
-            'pageTitle' => 'Incidents - Dashboard',
-            'incidents' => $incidents,
+             'pageTitle' => 'Incidents - Dashboard',
```
Comment (GrahamCampbell) [inline src/Http/Controllers/DashIncidentController.php]: ?

File: src/Http/Controllers/DashIncidentController.php [hunk 2]
```diff
@@ -89,4 +89,35 @@ public function deleteIncidentAction(Incident $incident)
...
+    /**
+     * Edit an incident.
+     *
+     * @param \CachetHQ\Cachet\Models\Incident $incident
+     *
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function editIncidentAction(Incident $incident)
+    {
+
```
Comment (GrahamCampbell) [inline src/Http/Controllers/DashIncidentController.php]: please remove the extra new line

**Comment by Crash--** [comment]
@GrahamCampbell done. But I don't know if StyleCI will trigger the fix.

**Comment by GrahamCampbell** [comment]
https://styleci.grahamjcampbell.co.uk/commits/831b433dd3a353b137c877e6cbcd53c252ee3edd/diff

**Comment by jbrooksuk** [comment]
Almost there @Crash-- :)

**Comment by GrahamCampbell** [comment]
I can fix this on merge tbh.

**Comment by GrahamCampbell** [comment]
Is everyone ok with this? // cc @cachethq/developers

**Comment by jbrooksuk** [comment]
Hold up.

@Crash-- can you remove the `.btn-warning` from the Edit button please.

**Comment by jbrooksuk** [comment]
Also, on the edit incident form, the submit button should be "Save Incident".

**Comment by joecohens** [comment]
I can confirm this is working, but has @jbrooksuk said, button should be .btn-default and "Save Incident".

**Comment by jbrooksuk** [comment]
@GrahamCampbell how do you push to an existing PR? I've checked it out, made the changes etc, but when I push, nothing happens.

**Comment by GrahamCampbell** [comment]
You can't push to someone else's fork. You'd have to merge this into a branch on this repo, then create another pull requests from this repo to this repo.

**Comment by jbrooksuk** [comment]
Mental.

**Comment by jbrooksuk** [comment]
Just pushed a fix for these changes.

**Comment by Crash--** [comment]
:+1: Thanks

**Pull request merged**